webapps/docs/config/host.xml

```diff
@@ -302,7 +302,7 @@
         placed in the <code>appBase</code> directory as web application
         archive (WAR) files to be unpacked into a corresponding disk directory
         structure, <code>false</code> to run such web applications directly
-        from a WAR file. See
+        from a WAR file. The flag's value defaults to <code>true</code>. See
```

Comment:
The default is...

Reply:
Shall I change this for all other occurences of the same wording?